Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.16.6 AS builder
+FROM golang:1.17.4 AS builder
```

I'm sorry, but 1.17.5 was just released. Would you mind updating the PR again?

```suggestion
FROM golang:1.17.5 AS builder
```

---

done! :)